test/README

```diff
@@ -10,19 +11,20 @@ saved (not mailed) and compared with the expected output for that
 feed/config pair.  For example, with a directory structure like::
 
   test
+  |-- data
```

Comment:
Why did you move all of the test configs and feed data into this `data/` directory? Is it something to do with the unittest test discovery maybe?

Reply:
Nothing special. I just wanted not to mix code with data.